web/src/components/CreateTestPlugins/Rest/steps/RequestDetails/RequestDetailsAuthInput/TypeInput.tsx

```diff
@@ -2,20 +2,18 @@ import {Form, FormInstance, Select} from 'antd';
 import {IRequestDetailsValues} from '../RequestDetails';
 import * as S from '../RequestDetails.styled';
 
-function methodNaming(method: string | null) {
-  switch (method) {
-    case 'apiKey':
-      return 'API Key';
-    case 'bearer':
-      return 'Bearer Token';
-    case 'basic':
-      return 'Basic Auth';
-    default:
-      return 'No Auth';
-  }
+const methodNamingMap: Record<string, string> = {
+  apiKey: 'API Key',
+  bearer: 'Bearer Token',
+  basic: 'Basic Auth',
+  none: 'No Auth',
+};
```

Comment:
If this is a constant you could probably have something like this (instead of `Record<string, string>`)
```ts
const methodNamingMap = {
  apiKey: 'API Key',
  bearer: 'Bearer Token',
  basic: 'Basic Auth',
  none: 'No Auth',
} as const;
```
With that, you create a typed object to those specific key-values